pipeline/metadata/4.3.yaml

```diff
@@ -589,3 +589,147 @@
     - ibmc
     - rgw
     - stage-1
+
+- name: "test-rgw-ms-lc-from-secondary"
+  suite: "suites/nautilus/rgw/tier-2_rgw_multisite_test-bucket-lifecycle-secondary-to-primary.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - schedule
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```

Comment:
We already have 5 test suites in `tier-2 & stage-2`... could we look at the next stages?

Reply:
For a scheduler job there is only one suite under tier-2 stage-2,
so included these suites under same stage.

Reply:
Thanks for clarifying... I missed the `schedule` tag.